crates/wasm-bindgen-cli-support/src/js.rs

```diff
@@ -656,6 +656,25 @@ impl<'a> Context<'a> {
         }
     }
 
+    fn expose_get_array_js_value_from_wasm(&mut self) {
+        if !self.exposed_globals.insert("get_array_js_value_from_wasm") {
+            return
+        }
+        self.expose_get_array_u8_from_wasm();
+        self.expose_get_object();
+        self.globals.push_str(&format!("
+                function getArrayJsValueFromWasm(ptr, len) {{
+                    const mem = getUint8Memory();
```

Comment:
This should be a u32 array, right?

Reply:
Whoops! Got lucky this worked every time I tested it. :-D